CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 ## Unreleased
 
+## v0.6.0
+
```

Comment:
Is it worth including the Bevy version here?

Normally we'd also include the MSRV, but I presume we're just inheriting that from Bevy?

Reply:
Yes, should include which version of bevy it is for as well as the date of the release. Even better would be to link to the changes and such like we do in all other Linebender releases.

Reply:
Would you be willing to make a different PR which implements this @simbleau?

Reply:
Done